styleguide/derek/_partials/_rug.ejs

```diff
@@ -1,12 +1,47 @@
 <!--[rug]-->
-<div class="container rug">
+<div class="container-fluid bg-blue-gray half-padding">
+<div class="container hidden-xs hidden-sm">
   <div class="row">
     <div class="col-sm-3">
-      <div class="col-sm-3">
-        <h5>We've got your back</h5>
-        <h4 class="text34">24x7x365</h4>
+      <h5 class="rug-subhead-medium no-margin">We've got your back</h5>
+      <h4 class="rug-subhead-large no-margin">24<small class="white">x</small>7<small class="white">x</small>365</h4>
     </div>
+    <div class="col-sm-3">
+      <div class="row">
+        <div class="col-lg-2 col-md-3">
+          <i class="fa fa-mobile fa-4x rug-icon"></i>
+        </div>
+        <div class="col-lg-10 col-md-9 rug-contact-link">
+          <a class="sales-phone rug-contact-link" href="tel:1-844-858-5963">1-844-858-5963</a> Sales<br/>
```

Comment:
Same thing here... just because text needs to be made an `a` doesn't mean it has to share the same color as all other anchors, nor does it mean we have to add in explicit overrides via class assignment or CSS. It's best to leave format/functionality elements alone and style by class as needed, but for now this will work.